tests/link/test_jax.py

```diff
@@ -1253,3 +1254,12 @@ def test_RandomStream():
     jax_res_2 = fn()
 
     assert np.array_equal(jax_res_1, jax_res_2)
+
+
+def test_Erf():
+    # @ericmjl: I am open to a much better implementation of this test!
+    x = scalar("x")
+    out = Erf(x)
+    fg = FunctionGraph(x, out)
```

Comment:
```suggestion
    fg = FunctionGraph([x], [out])
```

I think the inputs have to be lists, but I am not sure. Does the test pass locally for you?